monitoring_service/test/test_state_db.py

```diff
@@ -3,7 +3,7 @@ def check_monitor_request(data_sqlite, request_json):
     fields_to_check = list(request_json.keys())
     fields_to_check.remove('balance_proof')
     fields_to_check.remove('monitor_address')
-    to_check = list(data_sqlite.values())[0]
+    to_check = data_sqlite
```

Comment:
no need to create a new variable

Reply:
Actually the whole function is not necessary, anymore. I switched to a plain assert below.